src/core/opamFilename.ml

```diff
@@ -266,6 +266,12 @@ let exists filename =
 let opt_file filename =
   if exists filename then Some filename else None
 
+let with_tmp_file fn =
```

Comment:
Should it be `tmp` as `with_tmp_dir` of `temp` as `temp_file`...

Reply:
`tmp` is better imo because it follows the convention from the other similar functions from the same module. aka. `temp` only for the base function (`mk_temp_dir` and `temp_file`) then `tmp` for derived functions (`with_tmp_dir`, `in_tmp_dir` and `with_tmp_dir_job`)